packages/blockchain-wallet-v4-frontend/src/modals/SimpleBuy/LinkedPaymentAccounts/Accounts/index.tsx

```diff
@@ -225,7 +230,10 @@ class Accounts extends PureComponent<InjectedFormProps<{}, Props> & Props> {
         card: card.card,
         type: 'USER_CARD',
         currency: card.currency,
-        limits: { min: '1000', max: '500000' }
+        limits: {
+          min: cardMethod?.value.limits.min,
```

Comment:
since cardMethod can be undefined, should you put something like `cardMethod?.value.limits.min || 1000` etc?